File: Employee.py
#!/usr/bin/python
import sys

class Employee(object):
  def set_name(self, value):
    self._name = value
  def get_name(self):
    return self._name
  def del_name(self):
    del self.name
  name = property(get_name, set_name, del_name, 'String')

  def set_salary(self, value):
    self._salary = value
  def get_salary(self):
    return self._salary
  def del_salary(self):
    del self.salary
  salary = property(get_salary, set_salary, del_salary, 'float')

  def __init__(self, name, salary):
    self.name = name
    self.salary = salary
    
  def __repr__(self):
    return "Name: %s, Salary: $%.2f" % (self.name, self.salary)

  def displayEmployee(self):
    print self

def main():
  emp1 = Employee("Bob", 75000)
  emp2 = Employee("Joe", 50000)
  emp1.displayEmployee()
  emp2.displayEmployee()
  sys.exit(0)
  
if __name__ == '__main__':
  main();

File: Employee.cs
namespace Example {
  using System;
  
  class Employee {
    // public string Name { get; protected set; }
    private string _name;
    public string Name {
      get { return _name; }
      protected set { _name = value; }
    }
    
    // public float Salary { get; protected set; }
    private float _salary;
    public float Salary {
      get { return _salary; }
      protected set { _salary = value; }
    }
     
    public Employee(String name, float salary) {
     Name = name;
     Salary = salary;
    }
    
    public void displayEmployee() {
     Console.WriteLine(String.Format("Name: {0}, Salary: ${1:f}", Name, Salary));
    }
    
    static void Main(String[] args) {
     Employee emp1 = new Employee("Bob", 75000f);
     Employee emp2 = new Employee("Joe", 50000f);
     emp1.displayEmployee();
     emp2.displayEmployee();
    }
  }
}